tests/unit/backend/utils.py

```diff
@@ -10,12 +10,7 @@ class BogusProjectParams:
     def get(cls, **kwargs):
         bogus_data = dict(
             status=ProjectStatus.EDIT.name,
```

Comment:
only provide the minimum parameters.

Reply:
Why?

Reply:
because we aren't verifying those fields in tests that use this. The fields are verified in a different test.